tests/Bridge/Symfony/Bundle/Command/DebugResourceCommandTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Bridge\Symfony\Bundle\Command;
+
+use ApiPlatform\Bridge\Symfony\Bundle\Command\DebugResourceCommand;
+use ApiPlatform\Core\Tests\ProphecyTrait;
+use ApiPlatform\Exception\ResourceClassNotFoundException;
+use ApiPlatform\Metadata\Resource\Factory\AttributesResourceMetadataCollectionFactory;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+
+class DebugResourceCommandTest extends TestCase
+{
+    use ProphecyTrait;
+
+    private function getCommandTester(CliDumper $cliDumper = null): CommandTester
+    {
+        $application = new Application();
+        $application->setCatchExceptions(false);
+        $application->setAutoExit(false);
+
+        $application->add(new DebugResourceCommand(new AttributesResourceMetadataCollectionFactory(), new VarCloner(), $cliDumper ?? new CliDumper()));
+
+        $command = $application->find('debug:api-resource');
+
+        return new CommandTester($command);
+    }
+
+    public function testDebugResource()
+    {
+        $varDumper = $this->prophesize(CliDumper::class);
```

Comment:
I think that it's not necessary to mock this class (don't mock what you don't own). You may just instantiate a `CliDumper` dumping to `/dev/null`.

This should do the trick.

```suggestion
        $varDumper = new CliDumper('Windows' === PHP_OS_FAMILY ? 'NUL' : '/dev/null');
```

Reply:
But we want to test that dump is called isn't that alright?

Reply:
We should at least mock the interface instead of a concretion in this case.

Reply:
Another option could be to set `php://memory` as output and to assert that something has been written in it.